docs/sections/papers/deita.md

```diff
@@ -0,0 +1,382 @@
+
+The DEITA authors assume that you have access to open-source datasets that fit your use case. This may not be the case entirely. But with open-source communities tackling many use cases, with projects such as [BLOOM](https://arxiv.org/pdf/2110.08207.pdf) or [AYA](https://cohere.com/research/aya), it's likely that your use case will be tackled at some point. Furthermore, you could generate your own instruction/response pairs with methods such as [self-generated instructions](https://aclanthology.org/2023.acl-long.754/) using distilabel. This tutorial assumes that we have a data pool with excessive samples for the project's cost constraint. In short, we aim to achieve adequate performance from fewer samples.
+
+The authors claim that the subsample size "correlates proportionally with the computation consumed in instruction tuning". Hence on a first approximation, reducing the sample size means reducing computation consumption and so the total development cost. Reproducing the paper notations, we will associate the budget m to a number of instruction/response pairs that you can set depending on your real budget.
+
+![image.png](https://github.com/argilla-io/distilabel/blob/docs/deita-tutorial/docs/assets/tutorials-assets/deita/datasets.png)
+
+To match the experimental set-up, dataset *X\_sota* is a meta-dataset combining major open-source datasets available to instruct-tune LLMs. This dataset is composed of [ShareGPT](https://huggingface.co/datasets/shibing624/sharegpt_gpt4) (58k instruction/response pairs), [UltraChat](https://huggingface.co/datasets/stingning/ultrachat) (105k instruction/response pairs) and [WizardLM](https://github.com/nlpxucan/WizardLM) (143k instruction/response pairs). It sums to more than 300k instruction/response pairs. We aim to reduce the final subsample to 6k instruction/response pairs.
+
+## Setup the notebook and packages
+
+Let's prepare our dependencies:
+
+```bash
+pip install "distilabel[openai,hf-transformers]>=1.0.0"
+pip install pynvml huggingface_hub argilla
+```
+
+Import distilabel:
+
+```python
+from distilabel.llm.huggingface.transformers import TransformersLLM
+from distilabel.llm.openai import OpenAILLM
+from distilabel.pipeline.local import Pipeline
+from distilabel.steps.conversation import ConversationTemplate
+from distilabel.steps.deita import DeitaFiltering
+from distilabel.steps.expand import ExpandColumns
+from distilabel.steps.generators.huggingface import LoadHubDataset
+from distilabel.steps.task.complexity_scorer import ComplexityScorer
+from distilabel.steps.task.evol_instruct.base import EvolInstruct
+from distilabel.steps.task.evol_quality.base import EvolQuality
+from distilabel.steps.task.generate_embeddings import GenerateEmbeddings
+from distilabel.steps.task.quality_scorer import QualityScorer
```

Comment:
The imports are now updated, so that `llm`->`llms`, and `task.tasks`; also now the imports are grouped under their parent module, so that the above now looks like:

```suggestion
from distilabel.llms import TransformersLLM, OpenAILLM
from distilabel.pipeline import Pipeline
from distilabel.steps import ConversationTemplate, DeitaFiltering, ExpandColumns, LoadHubDataset
from distilabel.steps.tasks import ComplexityScorer, EvolInstruct, EvolQuality, GenerateEmbeddings, QualityScorer
```